03/app.js

```diff
@@ -1,10 +1,14 @@
 export default function randomNumber(min, max) {
-    if (typeof min !== 'number' || typeof max !== 'number') {
-        throw new Error('Property has to be a number')
+    if (typeof min !== 'number') {
+        throw new Error('Property "min" has to be a number')
+    }
+
+    if (typeof max !== 'number') {
```

Comment:
Tutaj warto jeszcze sprawdzić czy nie jest `NaN` bo to też `number`, ale raczej chcemy wykluczyć tą wartość ;)